languages-and-frameworks/wordpress.html.markerb

```diff
@@ -0,0 +1,160 @@
+---
+title: Run a Wordpress app
+layout: language-and-framework-docs
+redirect_from: /docs/getting-started/wordpress/
+---
+
+Setting up a wordpress application on Fly.io is quite simple. In this guide, we'll learn just how simple it is!
+
+There's one thing you need before setting up a wordpress application: a MySQL database. You can find out how to get one set up on Fly.io in [this handy guide](https://fly.io/docs/app-guides/mysql-on-fly/#main-content-start).
+Make sure you remember the hostname (this will be `<your-mysql-app-name>.internal`), the database name and the database credentials.
+
+Ready? Let's get started!
+
+## Launch a new wordpress application
+
+To launch a new wordpress application, we can use the [official wordpress docker image](https://hub.docker.com/_/wordpress). To do so, run the following command:
+```cmd
+flyctl launch -i wordpress --no-deploy
+```
+```output
+Using image wordpress
+Creating app in /Users/name/test/my-app-name
+We're about to launch your app on Fly.io. Here's what you're getting:
+
+Organization: MyOrg                  (fly launch defaults to the personal org)
+Name:         my-app-name            (derived from your directory name)
+Region:       Secaucus, NJ (US)      (this is the fastest region for you)
+App Machines: shared-cpu-1x, 1GB RAM (most apps need about 1GB of RAM)
+Postgres:     <none>                 (not requested)
+Redis:        <none>                 (not requested)
+
+? Do you want to tweak these settings before proceeding? Yes
+```
+
+Type `n` if you're happy with the defaults listed, otherwise type `y` to open the Fly Launch web page and edit your settings, including:
+
+* Name: Keep the default app name or enter your own.
```

Comment:
Maybe add something like, The name will be part of URL for your app, unless you use a custom domain.